beacon-chain/rpc/prysm/v1alpha1/validator/proposer_bellatrix.go

```diff
@@ -120,6 +123,22 @@ func (vs *Server) getPayloadHeaderFromBuilder(ctx context.Context, slot types.Sl
 	if err != nil {
 		return nil, err
 	}
+
+	v := bid.Message.Value
```

Comment:
is it possible to get a message without a value here? if it's external it might be nil maybe safer for a check here

Reply:
Good point, thanks!